### PR TITLE
VS can now run dolphinQt directly without DLL errors

### PR DESCRIPTION
**Pull request by rukai**
I solved it by setting the Local Windows Debugger command to "$(BinaryOutputDir)$(TargetFileName)"

---
This change is [Reviewable](https://reviewable.io/reviews/dolphin-emu/dolphin/3913)

### CODE REVIEW

**Comment by JosJuice** [comment]
Aren't .vcxproj.user files supposed to be specific to each user?

**Comment by rukai** [comment]
from [this documentation](https://msdn.microsoft.com/en-us/library/ee862524.aspx)
"A user file (.vcxproj.user) stores user-specific properties, for example, debugging and deployment settings. The vcxproj.user file applies to all projects for a particular user."
So, yes it is supposed to be specific to each user.

For some reason though, configuration for the command to run for launching the project is stored in there. So dolphinQt needs to provide its own.

File: Source/Core/DolphinQt2/DolphinQt2.vcxproj.user [hunk 1]
```diff
@@ -0,0 +1,11 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
```
Comment (shuffle2) [inline Source/Core/DolphinQt2/DolphinQt2.vcxproj.user]: these statements are equivalent

File: .gitignore [hunk 2]
```diff
@@ -15,7 +15,6 @@ Source/Core/Common/scmrev.h
 *.opensdf
 *.sdf
 *.suo
-*.vcxproj.user
```
Comment (shuffle2) [inline .gitignore]: please leave this

**Comment by rukai** [comment]
*[Source/Core/DolphinQt2/DolphinQt2.vcxproj.user, line 3 \[r1\]](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913#-KKkl0JBD3-yblpaEFKY-r1-3:-KKmnOi5B5kbTR9jNzYS:1691252438) ([raw file](https://github.com/dolphin-emu/dolphin/blob/d45b486e1aa610b7f388229768a471b667f47f34/Source/Core/DolphinQt2/DolphinQt2.vcxproj.user#L3)):*
<details><summary><i>Previously, shuffle2 wrote…</i></summary>
> these statements are equivalent
</details>
This file was generated by Visual Studio. Do you want me to manually edit it? I think its best to just let Visual Studio do what it wants.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913)*

**Comment by phire** [comment]
Review status: 0 of 2 files reviewed at latest revision, 1 unresolved discussion.

---

*[Source/Core/DolphinQt2/DolphinQt2.vcxproj.user, line 3 \[r1\]](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913#-KKkl0JBD3-yblpaEFKY-r1-3:-KKmnti9RNkPSl8RAAK6:-1946809866) ([raw file](https://github.com/dolphin-emu/dolphin/blob/d45b486e1aa610b7f388229768a471b667f47f34/Source/Core/DolphinQt2/DolphinQt2.vcxproj.user#L3)):*
<details><summary><i>Previously, rukai (Lucas Kent) wrote…</i></summary>
> This file was generated by Visual Studio. Do you want me to manually edit it? I think its best to just let Visual Studio do what it wants.
</details>
All our visual studio build files are hand-edited (and occasionally hand generated). The auto generated stuff is slightly too wordy.

Delete the second PropertyGroup and then Remove the Condition from the first.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913)*

**Comment by rukai** [comment]
*[Source/Core/DolphinQt2/DolphinQt2.vcxproj.user, line 3 \[r1\]](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913#-KKkl0JBD3-yblpaEFKY-r1-3:-KKn-2AxiYmHXAnC4lqz:-499146009) ([raw file](https://github.com/dolphin-emu/dolphin/blob/d45b486e1aa610b7f388229768a471b667f47f34/Source/Core/DolphinQt2/DolphinQt2.vcxproj.user#L3)):*
<details><summary><i>Previously, phire (Scott Mansell) wrote…</i></summary>
> All our visual studio build files are hand-edited (and occasionally hand generated). The auto generated stuff is slightly too wordy.
> 
> Delete the second PropertyGroup and then Remove the Condition from the first.
</details>
Done.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913)*

**Comment by BhaaLseN** [comment]
Not really convinced this is _the_ solution; *.user files shouldn't be in the repo. But I don't really know what else could there be, so meh. Looking at Unreal Engine, they generate .vcxproj and .user, but they never distribute it.
+/-0 on this, but I'd honestly think that Microsoft forgot something when this is really something every user has to set. Or it is a Qt issue and we're doing something wrong.

**Comment by rukai** [comment]
https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinWX/DolphinWX.vcxproj.user It is the solution used by dolphinwx.

---







*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913#-:-KKvaWXUaVTEPdlukPF2:-1809059910)*

**Comment by Parlane** [comment]
@BhaaLseN I asked @shuffle2 about using the user solution file, he assures me this is the way to do it.

**Comment by Parlane** [comment]
Reviewed 1 of 3 files at r1, 1 of 2 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/dolphin-emu/dolphin/3913)*
